docs/blog/_posts/2021-02-17-scala3-rc1.md

```diff
@@ -0,0 +1,242 @@
+
+<!--more-->
+# Allow leading context parameters in extension methods
+Type parameters on extensions can now be combined with type parameters on the methods themselves. E.g.:
+
+```scala
+List(1, 2, 3).second[Int]
+extension [T](xs: List[T])
+   def def sumBy[B](f: A => B)(using Numeric[B]): B = ...
+```
+
+Type arguments matching method type parameters are passed as usual:
+
+```scala
+List("a", "bb", "ccc").sumBy[Int](_.length)
+```
+By contrast, type arguments matching type parameters following `extension` can be passed
+only if the method is referenced as a regular method:
+
+```scala
+sumBy[String](List("a", "bb", "ccc"))(_.length)
+```
+
+Or, when passing both type arguments:
+
+```scala
+sumBy[String](List("a", "bb", "ccc"))[Int](_.length)
+```
+
+For discussion, see [PR #10940](https://github.com/lampepfl/dotty/pull/10940). For more information about the extension methods, see [documentation](https://dotty.epfl.ch/docs/reference/contextual/extension-methods.html).
+
+# New `import` syntax
+The following are the changes to the `import` syntax made in this release.
+
+Wildcard import `_` is replaced by `*`. The motivation is that the majority of other languages use `*`. For example:
+
+```scala
+import scala.annotation.*  // imports everything in the annotation package
+```
+
+Renaming operator `=>` is replaced by a soft keyword `as`. `as` is also allowed outside braces. For example:
+
+```scala
+import scala.collection.mutable as mut
+import NumPy as np
+```
+
+For the details and discussion, see [PR #11244](https://github.com/lampepfl/dotty/pull/11244). Read more about this change in the [documentation](https://dotty.epfl.ch/docs/reference/changed-features/imports.html).
+
+# Use `uninitialized` for wildcard initializers
+An obscure use of `_` occurs in var definitions:
+
+```scala
+var x: T = _
+```
+
+It defines a concrete variable x without an initial value, or rather the default initial value that the JVM assigns to object fields. It can only be used in a class or object, not to initialize a local variable.
+
+We came up with an arguably better way to express this idiom: the special `uninitialized` value in the `scala.compiletime` object. To get an uninitialized field, you now write:
+
+```scala
+import scala.compiletime.uninitialized
+
+var x: A = uninitialized
+```
+
+This way expresses the intent of the idiom in a more verbose and easy to read way than simply writing an underscore.
+
+For discussion, see [PR #11231](https://github.com/lampepfl/dotty/pull/11231), and the [documentation](https://dotty.epfl.ch/docs/reference/dropped-features/wildcard-init.html) is available on our website.
+
+# Allow `with` after class
+[PR #11157](https://github.com/lampepfl/dotty/pull/11157) brings an syntactic change whereby you have to write `with` instead of `:` to signify the start of the body of a class definition. For example:
+
+```scala
+object Foo with  // Was previously `object Foo:`
+  val x = 10
+```
+
```

Comment:
Add:

# Use `*` for vararg splices.

[PR #11240] changed the syntax of vararg splices in patterns and function arguments. The new syntax uses a postfix `*`, instead of `: _*`, analogously to how a vararg parameter is declared.